sql/schema.sql

```diff
@@ -22,6 +22,28 @@ CREATE TABLE music_albums (
     CONSTRAINT fk_author FOREIGN KEY(author_id) REFERENCES authors(id)
 );
 
+DROP TABLE IF EXISTS author;
+CREATE TABLE author (
+    id SERIAL PRIMARY KEY,
+    first_name VARCHAR(100) NOT NULL
+    last_name VARCHAR(100) NOT NULL
+)
+
+DROP TABLE IF EXISTS  game;
+CREATE TABLE game(
+    id SERIAL PRIMARY KEY,
+    genre INT,
+    author INT,
+    label INT,
```

Comment:
I suggest adding _id at the end of each name. Like: `genre_id` to maintain the relation with the other tables.